### DISCUSSION TITLE
Is z-push still under active development? Whats it's status?

### DISCUSSION THREAD

**Discussion by Danrancan** [Z-Hub/Z-Push#19] [General]
Is Z-Push still under active development or has it been discontinued? It seems like development ended in 2021, but I see commits from 2022 in the github repo. What is the status of z-push, and if it is discontinued, are their any viable alternatives?

**Comment by matidau** [comment]
Yep it is active. Last commit was last week to the develop branch. Do you mind if I move this issue to the discussions?

**Comment by Danrancan** [comment]
Thank you for your answer. Sorry I'm still a noob and didn't see the conversations section. Please move this if you want.
